File: 백준/Bronze/11721. 열 개씩 끊어 출력하기/열 개씩 끊어 출력하기.py
N = input() # 주어진 단어 str
K = len(N) # 주어진 단어 길이
text = [] # 빈 리스트

while True:
    if K == 0: # 주어진 단어 소진 시 그냥 출력 후 종료
        print(''.join(text)) 
        break
    for i in N:
        text.append(i) # 빈 리스트에 단어 한개씩 넣기
        if len(text) == 10: # 단어 10개 채워질때마다 출력
            print(''.join(text))
            text = [] # 리스트 초기화
        K -= 1 # 단어길이 줄이기

File: 백준/Bronze/11721. 열 개씩 끊어 출력하기/열 개씩 끊어 출력하기.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString();

let sentence = "";
let results = [];
for (let i = 0; i < input.length; i++) {
  if (sentence.length === 10) {
    results.push(sentence);
    sentence = input[i];
  } else {
    sentence += input[i];
  }
}
results.push(sentence);
for (let j = 0; j < results.length; j++) {
  console.log(results[j]);
}
